src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb

```diff
@@ -2091,7 +2095,9 @@ ProduceBoundNode:
 
                     Debug.Assert(argumentNames(i) Is Nothing OrElse argumentNames(i).Length > 0)
 
-                    If argumentNames(i) Is Nothing Then
+                    If argumentNames(i) Is Nothing AndAlso
+                        Not Me.Compilation.LanguageVersion.AllowNonTrailingNamedArguments() Then
```

Comment:
>Not Me.Compilation.LanguageVersion.AllowNonTrailingNamedArguments() [](start = 24, length = 67)

This change is unexpected because we don't have corresponding change in OverloadResolution. If it makes a difference, this is likely an indication of a problem.